app/src/main/java/org/launchcode/g7/bookcollections/ItemListFragment.java

```diff
@@ -117,6 +117,6 @@ public void onDetach()
     public interface OnListFragmentInteractionListener
     {
         // TODO: Update argument type and name
-        void onListFragmentInteraction(Object item);
+        void onListFragmentInteraction(Object item,int position);
```

Comment:
In order for the event to send the necessary parameters, the corresponding listener interface declaration (what you see here) had to be changed to include the new parameter.